DayPicker/tests/DayPicker-specs.js

```diff
@@ -1,40 +1,46 @@
 import ilib from '@enact/i18n';
-import {fireEvent, render} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import DayPicker, {getSelectedDayString} from '../DayPicker';
 
 describe('DayPicker', () => {
 
 	test('should not select any item when there is no passed prop `selected`', () => {
-		const {getAllByRole} = render(<DayPicker />);
+		render(<DayPicker />);
 
-		expect(getAllByRole('checkbox', {checked: false}));
+		const actual = screen.getAllByRole('checkbox', {checked: false});
```

Comment:
actually, this is a false positive expectation.. Now it checks that there are items with selected=false, instead of checking that there is no item with selected=true, as the title mentiones.